minecraft/protocol/packet/npc_dialogue.go

```diff
@@ -0,0 +1,49 @@
+package packet
+
+import "github.com/sandertv/gophertunnel/minecraft/protocol"
+
+const (
+	ActionTypeOpen int32 = iota
+	ActionTypeClose
```

Comment:
I think it might be a good idea to rename these slightly to make it clear these constants are for the NPC dialogues.

Reply:
Renamed.